lib/components/note-preview/index.tsx

```diff
@@ -29,9 +29,13 @@ type DispatchProps = {
   openNote: (noteId: T.EntityId) => any;
 };
 
-type Props = OwnProps & StateProps & DispatchProps;
+type Props = OwnProps &
+  StateProps &
+  DispatchProps &
+  React.HTMLProps<HTMLDivElement>;
 
 export const NotePreview: FunctionComponent<Props> = ({
+  'aria-hidden': ariaHidden,
```

Comment:
Seems odd to have the prop be `aria-hidden` just to set it to ariaHidden to set it to the attribute `aria-hidden`

Reply:
Agreed. It's a little odd. The cause for this originates from https://github.com/Automattic/simplenote-electron/pull/2707#discussion_r591678420. If we decided to replace kebab-case for camelcase there, this will go away.